css/css-align-3/distribution-values/space-evenly-001.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html lang=en>
+  <meta charset=utf-8>
+  <title>CSS Box Alignment: space-evenly & flexbox with single item</title>
+  <link rel="author" title="Florian Rivoal" href="https://florian.rivoal.net">
+  <meta name=flags content="">
```

Comment:
I don't know about the new linter, but I believe the old one wanted it, to prove that you had considered it and decided you didn't need any flag, and not just forgotten about it.